governance/votes/Executive vote - January 24, 2024.md

```diff
@@ -0,0 +1,143 @@
+- The `WBTC-B` [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be increased by 0.91%, from 5.54% to **6.45%**.
+
+### Decrease PSM-PAX-A Debt Ceiling
+
+**Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445/3)\
+**Proposal:** [Forum post "Stability Scope Parameter Changes #8"](https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445)
+
+If this executive proposal passes, in line with the recent removal of `PSM-PAX-A` from the Cash Stablecoin list in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#7-2-1-3-1a), the following actions will be performed:
+
+- The [Debt Ceiling (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to **zero**.
+- The vault will be removed from the [Debt Ceiling Instant Access Module (AutoLine)](https://manual.makerdao.com/module-index/module-dciam).
+
+### RETH-A Offboarding Parameters Finalization
+
+**Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445/3)\
+**Proposal:** [Forum post "Stability Scope Parameter Changes #8"](https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445)
+
+ The offboarding of `RETH-A` will be completed by enacting the following parameter changes, if this executive proposal passes:
+
+- Reduce [Liquidation Penalty (`chop`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-penalty) by 13% from 13% to **0%**.
+- Reduce [Flat Kick Incentive (`tip`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) by 250 DAI from 250 DAI to **0 DAI**.
+- Reduce [Proportional Kick Incentive (`chip`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-proportional-kick-incentive) by 0.1% from 0.1% to **0%**.
+- Increase [Liquidation Ratio (`mat`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) by 9,850% from 150% to **10,000%**.
+
+### SBE Parameter Changes
+
+ **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441/2)\
+ **Proposal:** [Forum post "Smart Burn Engine - Transaction Analysis & Parameter Reconfiguration Update #4"](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441)
+
+ If this executive proposal passes, following the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SBE parameters will be changed:
+
+- The [`bump` (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) parameter will be increased by 20,000, from 30,000 to **50,000 DAI**.
+- The [`hop` (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) parameter will be increased by 10,512, from 15,768 to **26,280 seconds**.
```

Comment:
Add units, see https://github.com/makerdao/community/pull/2534/files#diff-05e20bcb70a29f4d9478bd6ea7f745722cabe1b590058c0a4a6102ec7e7f5635R72-R75 where units are included for each line item

```suggestion
- The [`bump` (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) parameter will be increased by 20,000 DAI, from 30,000 DAI to **50,000 DAI**.
- The [`hop` (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) parameter will be increased by 10,512 seconds , from 15,768 seconds to **26,280 seconds**.
```